pkg/formula/input/prompt/prompt.go

```diff
@@ -224,27 +222,23 @@ func (in InputManager) loadInputValList(items []string, input formula.Input) (st
 func (in InputManager) loadItems(input formula.Input, formulaPath string) ([]string, error) {
 	if input.Cache.Active {
```

Comment:
Could you simplify this function as suggested below?

```suggestion
	if !input.Cache.Active {
		return input.Items, nil
	}
	 
	cachePath := fmt.Sprintf(CachePattern, formulaPath, strings.ToUpper(input.Name))
	fileBytes, err := ioutil.ReadFile(cachePath)
	if err != nil {
		itemsBytes, err := json.Marshal(input.Items)
		if err != nil {
			return nil, err
		}
	
		if err = ioutil.WriteFile(cachePath, itemsBytes, os.ModePerm); err != nil {
			return nil, err
		}
		
		return input.Items, nil
	}
		
	var items []string
	if err := json.Unmarshal(fileBytes, &items); err != nil {
		return nil, err
	}

	return items, nil
```